### PR TITLE
feat: projects fullscreen view

### PR DESCRIPTION
**Pull request by 1emu**
image

image

image

image

image

### CODE REVIEW

File: src/components/Projects/ProjectViewFundingSection.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+.ProjectViewFundingSection__FundingBox {
+  display: flex;
+  padding: 16px;
+  flex-direction: column;
+  gap: 12px;
+  border-radius: 8px;
+  border: 1px solid var(--black-300);
+  background: var(--white-900);
+  box-shadow: 0 0 10px 0 rgba(115, 110, 125, 0.15);
```
Comment (ncomerci) [inline src/components/Projects/ProjectViewFundingSection.css:14]: `--alpha-black-200` is the closest color

**Review by ncomerci — APPROVED** [review]

File: src/components/Projects/ProjectView.tsx [hunk 2]
```diff
@@ -53,28 +53,40 @@ function ProjectView({ project, onClose, isFullscreen = false }: Props) {
...
-      </BoxTabs>
-      <div
-        className={classNames(
-          'ProjectView__ContentContainer',
-          isFullscreen && 'ProjectView__ContentContainer--fullscreen'
-        )}
-      >
+  return (
+    <div className="ProjectView">
+      {project && <ProjectViewTitle project={project} onClose={onClose} />}
```
Comment (andyesp) [inline src/components/Projects/ProjectView.tsx:59]: What do we show if there is no project?

File: src/components/Projects/ProjectViewFundingSection.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+function getVestingLinkLabel(index: number, thereIsMoreThanOnePastVesting: boolean) {
+  return `${index === 0 ? 'Current' : 'Past'} vesting ${thereIsMoreThanOnePastVesting && index > 0 ? `${index}` : ''}`
+}
+
+function ProjectViewFundingSection({ project, compact, className }: Props) {
+  const { vesting_addresses, funding } = project
+  const thereIsMoreThanOnePastVesting = vesting_addresses.length > 2
+
+  const t = useFormatMessage()
+  return project.funding ? (
```
Comment (andyesp) [inline src/components/Projects/ProjectViewFundingSection.tsx:34]: I'd do here
```
if(!project.funding) {
  return null
}

... the rest of the component
```

File: src/components/Proposal/View/ProjectViewLink.tsx [hunk 4]
```diff
@@ -3,40 +3,41 @@ import { Button } from 'decentraland-ui/dist/components/Button/Button'
...
   projectStatus: ProjectStatus
   isGrantee: boolean
-  onClick?: () => void
+  projectUrl: string
 }
 
-function ProjectSheetLink({ projectStatus, isGrantee, onClick }: Props) {
+function ProjectViewLink({ projectStatus, isGrantee, projectUrl }: Props) {
   const t = useFormatMessage()
+  console.log(projectUrl)
```
Comment (andyesp) [inline src/components/Proposal/View/ProjectViewLink.tsx:21]: console.log

**Review by andyesp — APPROVED** [review]

File: src/components/Projects/ProjectView.tsx (hunk 2 above)
Comment (1emu) [reply src/components/Projects/ProjectView.tsx:59]: image

**Pull request merged**